DataFormats/Detectors/TRD/include/DataFormatsTRD/NoiseCalibration.h

```diff
@@ -52,13 +53,23 @@ class NoiseStatusMCM
   bool getIsNoisy(int mcmIdxGlb) const { return mNoiseFlag.test(mcmIdxGlb); }
   auto getNumberOfNoisyMCMs() const { return mNoiseFlag.count(); }
   bool isTrackletFromNoisyMCM(const Tracklet64& trklt) const { return getIsNoisy(trklt.getHCID(), trklt.getROB(), trklt.getMCM()); }
+  bool isDigitFromNoisyMCM(const Digit& d) const { return getIsNoisy(d.getHCId(), d.getROB(), d.getMCM()); }
 
  private:
```

Comment:
Don't you need to update ClassDefNV?

Reply:
That is only required if the class layout changes. Since I am not adding/removing a class member its not needed